src/app/state/form/selectors/form.selector.ts

```diff
@@ -0,0 +1,21 @@
+import {createSelector} from '@ngrx/store';
+import {selectParameterGroupStationMappings} from '../../parameter-station-mapping/selectors/parameter-group-station-mapping.selector';
+import {selectCurrentStationState} from '../../stations/selectors/station.selector';
+import {formFeature} from '../reducers/form.reducer';
+
+export const selectCollectionsForSelectedStation = createSelector(
+  formFeature.selectSelectedStationId,
+  formFeature.selectSelectedParameterGroupId,
+  selectCurrentStationState,
+  selectParameterGroupStationMappings,
+  (stationId, selectedParameterGroupId, stationState, stationGroupMapping) => {
```

Comment:
```suggestion
  (stationId, selectedParameterGroupId, {stations}, stationGroupMappings): Collection[] => {
```
* `{stations}` instead of the whole state
* plural: stationGroupMapping**s**
* return type `Collection[]`